lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -260,22 +262,45 @@ bool AsyncPipelinedClient::deactivate_all_flows(const std::string& imsi) {
   return true;
 }
 
```

Comment:
in order to make sure we differentiate between the two cases of remove all the rules + any default installed rules(which happen at termination) vs remove a specific set or rules (which may happen during a regular rule removal, but not termination), I decided to be explicit and separate the call depending if it is for termination or not.

Reply:
sg~!